pkgs/top-level/python-packages.nix

```
@@ -2930,6 +2932,8 @@ in {
 
   python-ctags3 = callPackage ../development/python-modules/python-ctags3 { };
 
+  python-lzo = callPackage ../development/python-modules/python-lzo { };
```

future proof against shadowing
```suggestion
  python-lzo = callPackage ../development/python-modules/python-lzo {
    inherit (pkgs) lzo;
  };
```